### ISSUE TITLE
Scrolling gets stuck on iOS Safari

### ISSUE THREAD

**Issue by lizzthabet** [tilt-dev/tilt.build#970]
[Chris Dary on Twitter](https://twitter.com/chrisdary/status/1454279664575332354) reports:
> just fyi y’all need to fix the scrolling in iOS safari for your marketing site. It gets stuck unless you pinch zoom out first

**Comment by lizzthabet** [comment]
@Chimney42 and I were able to reproduce on an iPhone running iOS 14 and through XCode Simulator.

To reproduce:
1. Go to `tilt.dev` in Safari.
2. Try to scroll the page immediately. The page will not scroll and seem stuck.
3. After waiting several seconds, try scrolling again. The page should scroll just fine. (Lian reported that it took longer for scrolling to start working if she tried to scroll a lot when the page first loaded. I did not see a difference on my end through XCode Simulator.)
4. Disable Javascript for Safari under Settings.
5. Go to `tilt.dev` again in Safari, either by refreshing the page or opening a new tab.
6. Try to scroll the page immediately. The page should scroll just fine.'

Pinch zooming in and out didn't make a difference in our testing. (It seems likely that after pinch zooming, enough time had passed that the page was scrollable.)
